components/calendly_v2/actions/get-event/get-event.mjs

```diff
@@ -14,21 +15,48 @@ export default {
     },
     event_uuid: {
       type: "string",
-      description: "The event's unique identifier.",
+      label: "Event UUID",
+      description:
+        "The event's unique identifier.",
+      optional: true,
+    },
+    event_url: {
+      type: "string",
+      label: "Event URL",
+      description: "The URL of the event to retrieve information about. If you are using a Calendly Source in the same workflow, you would use *{{steps.trigger.event.payload.event}}*.",
+      optional: true,
     },
   },
+
   async run({ $ }) {
-  //See the API docs: https://developer.calendly.com/docs/api-docs/reference/calendly-api/openapi.yaml/paths/~1scheduled_events~1%7Buuid%7D/get
+    function getEventUuidFromUrl(event_url) {
```

Comment:
Please move this function into methods, [see a working example here](https://github.com/PipedreamHQ/pipedream/blob/master/components/badger_maps/actions/get-account/get-account.mjs) and see the Pipedream's [documentation here](https://pipedream.com/docs/components/api/#component-api)